frontend/src/v5/ui/routes/dashboard/projects/drawings/drawingsList/drawingsListItem/drawingsListItem.component.tsx

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
"Actions" text uses a different colour, also, that is missing in containers/fed. Should we create an issue to add it there as well?
![image](https://github.com/3drepo/3drepo.io/assets/43547703/5358b886-1407-4ef8-b065-85a04bc7e39a)